libusbdrivers/src/services.h

```diff
@@ -21,65 +21,64 @@ void otg_irq(void);
 extern ps_malloc_ops_t *ps_malloc_ops;
 static inline void *usb_malloc(size_t size)
 {
-	int ret;
+    int ret;
 
-	if (ps_malloc_ops && ps_malloc_ops->calloc) {
-		void *ptr;
-		ret = ps_calloc(ps_malloc_ops, 1, size, &ptr);
-		if (!ret) {
-			ZF_LOGF("Malloc error\n");
-		}
-		return ptr;
-	} else {
-		return calloc(1, size);
-	}
+    if (ps_malloc_ops && ps_malloc_ops->calloc) {
+        void *ptr;
+        ret = ps_calloc(ps_malloc_ops, 1, size, &ptr);
+        if (ret) {
+            ZF_LOGF("Malloc error\n");
```

Comment:
If `ret` holds an an error code, then we could also print it here. Otherwise the error code is lost and it's hard to debug what actually failed.